src/functions/welcome/welcome.go

```diff
@@ -0,0 +1,66 @@
+package welcome
+
+import (
+	"encoding/json"
+	"fmt"
+	"html"
+	"net/http"
+	"strings"
+	"os"
+
+	"github.com/bwmarrin/discordgo"
+)
+
+// Data struct from Pubsub
+type PubSubMessage struct {
+	Data []byte `json:"data"`
+}
+
+type Payload struct {
+	UserId  string `json:"user_id"`
+	Username string `json:"username"`
+	ServerId string `json:"server_id"`
+	ServerName string `json:"server_name"`
+	AvatarUrl string `json:"avatar_url"`
+	ChannelId string `json:"channel_id"`
+}
+
+func GreetingNewMember(w http.ResponseWriter, r *http.Request) {
+	//Parse body to get Payload
+	var payload = Payload{}
+	err := json.NewDecoder(r.Body).Decode(&payload)
+
+	if err != nil {
+		panic(err)
+	}
+
+	if len(payload.UserId) == 0 
+	or len(payload.Username) == 0 
+	or len(payload.ServerName) == 0 
+	or len(payload.AvatarUrl) == 0 {
```

Comment:
`or` doesn't exist in Go, use `||`
https://www.tutorialkart.com/golang-tutorial/golang-or/

Reply:
Alright! it's fixed